### DISCUSSION TITLE
[FF7] Audio Issues CAMS-R

### DISCUSSION THREAD

**Discussion by CarlucciMods** [julianxhokaxhiu/FFNx#371] [Q&A] [answered]
To the Entire Team,

Thank you so much for all your hard work and dedication. 

My issue lies with the CAMS-R Mod. I have attached my mod.xml and even one of the config.toml that i been using to test.

My mod is a music shuffler where users can pick which title they want to activate and then choose from a choice in genre's.  It worked yesterday, but now it doesn't seem to work. I am on the latest driver. 

I was told there has to be an originally named songs in the folder before it will reroute.
for example mine are bat.ogg, fan2.ogg and a chu.ogg... They're blank files.

[CAMS-R.zip](https://github.com/julianxhokaxhiu/FFNx/files/7998059/CAMS-R.zip)

**Comment by julianxhokaxhiu** [comment, marked as answer]
This issue is unrelated to FFNx. Please consider approaching 7th relative channels for support. Thank you.
